e2e-tests/cypress/support/appellant-submission-check-your-answers/confirmCheckYourAnswersPage.js

```diff
@@ -1,4 +1,4 @@
 module.exports = () => {
-  cy.url().should('include', '/appellant-submission/check-answers');
+  cy.url().should('include', '/appeal-householder-decision/check-answers');
```

Comment:
please can the dir here be updated to reflect `appeal-householder-decision` - thanks